sdk/storage/.dict.txt

```diff
@@ -20,3 +20,7 @@ uncommittedblobs
 policyid
 ruleid
 westus
+testblob1
+testblob2
+testblob3
+testblob4
```

Comment:
Tip: if you only refer to these in a file, you can just add a comment like:

```rust
// cspell:ignore testblob1 testblob2 testblob3
```

Just helps localize changes. Applies from that line forward.

Reply:
Good tidbit, thanks! I will leave it in the dict since we will be adding more tests in different clients that will require multiple created blobs.